config/src/config/node_config.rs

```diff
@@ -52,6 +52,8 @@ pub struct NodeConfig {
     #[serde(default)]
     pub indexer_grpc: IndexerGrpcConfig,
     #[serde(default)]
+    pub indexer_table_info: IndexerTableInfoConfig,
```

Comment:
Silly question: do we need this to be a high-level config? Or could it live inside the existing `indexer` or `indexer_grpc` configs?

A quick skim of the sanitizer for `IndexerGrpcConfig` suggests that they should both be enabled/disabled together, and thus they should maybe just live together?

Reply:
this needs to be a high level config and its not coupled with `indexer_grpc` cuz it doesn't depend on that, but `indexer_grpc` depends on this: if you look at this [sanitize](https://github.com/aptos-labs/aptos-core/blob/966b4b1e2269ac4576335c9d615c4d3d6bea1934/config/src/config/indexer_grpc_config.rs#L60-L80) logic. so its still possible to enable this standalone config without enabling `indexer_grpc`, but just not the other way around.